test/Fhir.Metrics.Tests/Data/ucum-tests.xml

```diff
@@ -1,415 +1,733 @@
+	<case id="1-297" unit="mm[Hg]" valid="true"/>
+	<case id="1-298" unit="mmol" valid="true"/>
+	<case id="1-299" unit="mmol/12h" valid="false" reason="not a valid unit (12h)"/>
+	<case id="1-300" unit="mmol/5h" valid="false" reason="not a valid unit (5h)"/>
+	<case id="1-301" unit="mmol/6h" valid="false" reason="not a valid unit (6h)"/>
+	<case id="1-302" unit="mmol/d" valid="true"/>
+	<case id="1-303" unit="mmol/g" valid="true"/>
+	<case id="1-304" unit="mmol/kg" valid="true"/>
+	<case id="1-305" unit="mmol/kg[H20]" valid="false" reason="not a valid unit (kg[H20])"/>
+	<case id="1-306" unit="mmol/L" valid="true"/>
+	<case id="1-307" unit="mmol/mmol" valid="true"/>
+	<case id="1-308" unit="mU/L" valid="true"/>
+	<case id="1-309" unit="ng/d" valid="true"/>
+	<case id="1-310" unit="ng/g" valid="true"/>
+	<case id="1-311" unit="ng/L" valid="true"/>
+	<case id="1-312" unit="ng/mL" valid="true"/>
+	<case id="1-313" unit="nmol/d" valid="true"/>
+	<case id="1-314" unit="nmol/g" valid="true"/>
+	<case id="1-315" unit="nmol/h/mL" valid="true"/>
+	<case id="1-316" unit="nmol/L" valid="true"/>
+	<case id="1-317" unit="nmol/mmol" valid="true"/>
+	<case id="1-318" unit="nmol/nmol" valid="true"/>
+	<case id="1-319" unit="pg" valid="true"/>
+	<case id="1-320" unit="pg/mL" valid="true"/>
+	<case id="1-321" unit="pmol/d" valid="true"/>
+	<case id="1-322" unit="pmol/g" valid="true"/>
+	<case id="1-323" unit="pmol/h/mg" valid="true"/>
+	<case id="1-324" unit="pmol/h/mL" valid="true"/>
+	<case id="1-325" unit="pmol/L" valid="true"/>
+	<case id="1-326" unit="pmol/mmol" valid="true"/>
+	<case id="1-327" unit="s" valid="true"/>
+	<case id="1-328" unit="U" valid="true"/>
+	<case id="1-329" unit="U/12h" valid="false" reason="not a valid unit (12h)"/>
+	<case id="1-330" unit="U/1h" valid="false" reason="not a valid unit (1h)"/>
+	<case id="1-331" unit="U/2h" valid="false" reason="not a valid unit (2h)"/>
+	<case id="1-332" unit="U/d" valid="true"/>
+	<case id="1-333" unit="U/g" valid="true"/>
+	<case id="1-334" unit="U/kg" valid="true"/>
+	<case id="1-335" unit="U/L" valid="true"/>
+	<case id="1-336" unit="U/mL" valid="true"/>
+	<case id="1-337" unit="u[IU]/mL" valid="true"/>
+	<case id="1-337a" unit="u[iIU]/mL" valid="false" reason="not a valid unit (iIU)"/>
+	<case id="1-338" unit="ug" valid="true"/>
+	<case id="1-339" unit="ug/d" valid="true"/>
+	<case id="1-340" unit="ug/g" valid="true"/>
+	<case id="1-341" unit="ug/L" valid="true"/>
+	<case id="1-342" unit="ug/mL" valid="true"/>
+	<case id="1-343" unit="um/s" valid="true"/>
+	<case id="1-344" unit="umol" valid="true"/>
+	<!-- <case id="1-345" unit="umol/2.h" valid="true"/> -->
+	<case id="1-346" unit="umol/d" valid="true"/>
+	<case id="1-347" unit="umol/g" valid="true"/>
+	<case id="1-348" unit="umol/L" valid="true"/>
+	<case id="1-349" unit="umol/mmol" valid="true"/>
+	<case id="1-350" unit="umol/umol" valid="true"/>
+	<case id="1-351" unit="wk" valid="true"/>
+       
+	<!-- 

```

Comment:
Because there are new tests (and comments) with the update to the latest UCUM tests, *and* comments by yourself, it is not distinguishable what was already a comment and what was added by you.
Suggest to mark our comments with 'Firely:'